pkg/payload/render.go

```diff
@@ -31,6 +35,35 @@ func Render(outputDir, releaseImage, clusterProfile string) error {
 		}
 	)
 
+	var requiredFeatureSet *string
```

Comment:
```suggestion
	requiredFeatureSet := ptr.To[string]("")
```

Reply:
deferring for now, in the rush to hit 4.17 branching.